packages/admin/src/pages/super-admin-dashboard/user/[id]/index.page.tsx

```diff
@@ -60,20 +61,44 @@ const UserPage = ({ user }: UserPageProps) => {
                     },
                     {
                       key: 'Roles',
-                      value: user.role?.label || 'Applicant',
-                      action: (
+                      value: user.role?.label || 'Blocked',
+                      action: user.role?.label ? (
                         <Link
                           href={`/super-admin-dashboard/user/${user.gapUserId}/change-roles`}
                         >
                           <a className="govuk-link">Change</a>
                         </Link>
+                      ) : (
+                        <></>
                       ),
                     },
                   ]}
                 />
                 <div className="govuk-button-group">
-                  <Button text="Delete user" isWarning />
-                  <Button text="Block user" isSecondary />
+                  <CustomLink
+                    href={`/super-admin-dashboard/user/${user.gapUserId}/delete-user`}
+                    customStyle="govuk-button govuk-button--warning"
+                    data-module="govuk-button"
+                  >
+                    Delete user
+                  </CustomLink>
```

Comment:
what's the difference between `<CustomLink>Text</CustomLink>` and `<Link><a>Text</a></Link>`? I see them used relatively interchangably